.github/workflows/automate_epinow2.yml

```diff
@@ -0,0 +1,111 @@
+name: "Automation"
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 13 * * 2"
+
+jobs:
+  generate-forecasts:
+    runs-on: ubuntu-20.04
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+    
+    - uses: r-lib/actions/setup-r@v1
+      with:
+        use-public-rspm: true
+
+    - name: Install system dependencies
+      run: sudo apt-get install libudunits2-dev libcurl4-openssl-dev libgdal-dev libv8-dev
+
+    - name: Install R dependencies
+      run: |
+       install.packages(c("fread", "glue"))
+      shell: Rscript {0}
+
+    - name: Generate forecasts
+      run: |
```

Comment:
What about just

```sh
TODAY=date --iso-8601
wget https://raw.githubusercontent.com/epiforecasts/europe-covid-forecast/master/submissions/rt-forecasts/$TODAY/$TODAY-epiforecasts-EpiNow2.csv
```

?

So we can get rid of the R step and shave off some running time.